src/map/utils/petutils.cpp

```diff
@@ -898,28 +900,28 @@ namespace petutils
 
         PPet->m_SpellListContainer = mobSpellList::GetMobSpellList(petData->spellList);
 
-        PPet->setModifier(Mod::SLASHRES, petData->slashres);
-        PPet->setModifier(Mod::PIERCERES, petData->pierceres);
-        PPet->setModifier(Mod::HTHRES, petData->hthres);
-        PPet->setModifier(Mod::IMPACTRES, petData->impactres);
-
-        PPet->setModifier(Mod::FIREDEF, petData->firedef);       // These are stored as floating percentages
-        PPet->setModifier(Mod::ICEDEF, petData->icedef);         // and need to be adjusted into modifier units.
-        PPet->setModifier(Mod::WINDDEF, petData->winddef);       // Higher DEF = lower damage.
-        PPet->setModifier(Mod::EARTHDEF, petData->earthdef);     // Negatives signify increased damage.
-        PPet->setModifier(Mod::THUNDERDEF, petData->thunderdef); // Positives signify reduced damage.
-        PPet->setModifier(Mod::WATERDEF, petData->waterdef);     // Ex: 125% damage would be 1.25, 50% damage would be 0.50
-        PPet->setModifier(Mod::LIGHTDEF, petData->lightdef);     // (1.25 - 1) * -1000 = -250 DEF
-        PPet->setModifier(Mod::DARKDEF, petData->darkdef);       // (0.50 - 1) * -1000 = 500 DEF
-
-        PPet->setModifier(Mod::FIRERES, petData->fireres);       // These are stored as floating percentages
-        PPet->setModifier(Mod::ICERES, petData->iceres);         // and need to be adjusted into modifier units.
-        PPet->setModifier(Mod::WINDRES, petData->windres);       // Higher RES = lower damage.
-        PPet->setModifier(Mod::EARTHRES, petData->earthres);     // Negatives signify lower resist chance.
-        PPet->setModifier(Mod::THUNDERRES, petData->thunderres); // Positives signify increased resist chance.
-        PPet->setModifier(Mod::WATERRES, petData->waterres);
-        PPet->setModifier(Mod::LIGHTRES, petData->lightres);
-        PPet->setModifier(Mod::DARKRES, petData->darkres);
+        PPet->setModifier(Mod::SLASH_SDT, petData->slash_sdt);
+        PPet->setModifier(Mod::PIERCE_SDT, petData->pierce_sdt);
+        PPet->setModifier(Mod::HTH_SDT, petData->hth_sdt);
+        PPet->setModifier(Mod::IMPACT_SDT, petData->impact_sdt);
+
+        PPet->setModifier(Mod::FIRE_SDT, petData->fire_sdt);       // These are stored as floating percentages
+        PPet->setModifier(Mod::ICE_SDT, petData->ice_sdt);         // and need to be adjusted into modifier units.
+        PPet->setModifier(Mod::WIND_SDT, petData->wind_sdt);       // Higher SDT = lower damage.
```

Comment:
That comment doesnt make sense?
Higher Res does mean lower damage, for A or B reason, but higher Specific Damage Taken... Should mean higer damage?

Reply:
in the sql end, its a % basically, 0.00 would mean no dmg, and 1.00 would mean full damage

but in the modifier is higher value = less damage, its inverted and stupid but I have not changed it yet - when the mod gets changed to be 1000 base instead of 256 that'll happen and the comment will be saner. was just to much to explain in line so its more a self reminder wtf the mod does.